full-service/src/db/transaction_log.rs

```diff
@@ -509,14 +517,13 @@ impl TransactionLogModel for TransactionLog {
             if let Some(recipient) = recipient_address {
                 let transaction_id = TransactionID::from(&tx_proposal.tx);
                 let tx = mc_util_serial::encode(&tx_proposal.tx);
+
                 // Create a TransactionLogs entry
                 let new_transaction_log = NewTransactionLog {
                     transaction_id_hex: &transaction_id.to_string(),
-                    account_id_hex: &account_id_hex.unwrap_or(""), /* Can be empty str if
-                                                                    * submitting an "unowned"
-                                                                    * proposal */
+                    account_id_hex, // Can be null if submitting an "unowned" proposal.
```

Comment:
We've previously introduced the concept that you must provide an account_id to log submitted, so this can actually be non-null